romancal/assign_wcs/pointing.py

```diff
@@ -1,7 +1,12 @@
+import logging
+
 import numpy as np
-from astropy.modeling.models import RotationSequence3D, Scale
+from astropy.modeling.models import Identity, RotationSequence3D, Scale, Shift
 from gwcs.geometry import CartesianToSpherical, SphericalToCartesian
 
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
```

Comment:
We don't do this elsewhere AFAIK; was this intentional?

Reply:
Missed this cut and paste error. Removing.